README.md

```diff
@@ -1,3 +1,135 @@
-# Wix build rules and macros for Bazel
+<h1>Wix build rules and macros for Bazel</h1>
+
+<h3 id="wix-logo">
+	<img src="assets/icons/bazel.svg" height="80" width="80">BUILD 
+	<img src="assets/icons/musa.svg" height="80" width="80">EXTEND
```

Comment:
musa icon is only internal to Wix. 
Probably better using official wix engineering logo.

Reply:
Replaced with official logo.